Riot/ViewController/RoomViewController.m

```diff
@@ -3724,25 +3723,33 @@ - (void)refreshMissedDiscussionsCount:(BOOL)force
             // Update the label background view frame
             CGRect frame = missedDiscussionsBadgeLabelBgView.frame;
             frame.size.width = round(missedDiscussionsBadgeLabel.frame.size.width + 18);
-            if (backButton && !backButton.title.length)
-            {
-                // Shift the badge on the left to be close the back icon
-                frame.origin.x = ([GBDeviceInfo deviceInfo].displayInfo.display > GBDeviceDisplay4Inch ? -35 : -25);
-            }
-            else
+            
+            if ([GBDeviceInfo deviceInfo].osVersion.major < 11)
```

Comment:
Why not using ```if (@available(iOS 11.0, *))``` as everywhere else in this PR?

Reply:
Because we only have to do something if iOS 11.0 is not available. This would mean an empty block:
`if (@available(iOS 11.0, *)) {} else {...}`